public/cli/lib/commands/ffmpeg.ts

```diff
@@ -8,6 +8,25 @@ import { paths } from '../path-constants';
 
 export async function combineFrames(settings: FfmpegSettings): Promise<void> {
   const executeAudioPath = await combineAudioIfNecessary(settings.audioFiles);
+  
+  // check if output file already exists:
+  var outputName = settings.outputName;
+  if (fs.existsSync(outputName)) {
+    // if they want to keep the file then modify the outputName
+    if (!settings.overwriteOutputFiles) {
+      // get a list of all the files in the directory,
+      var listAllFiles = fs.readdirSync(settings.imagesPath, 'utf8');
+      // get a count of the ones that have a similar start name
+      var nameParts = path.parse(outputName);
+      var matchingFiles = (listAllFiles ??[]).filter((f) => f.indexOf(nameParts.name) == 0);
```

Comment:
In JS you should no longer use `==` for comparison because it coerces and its not always clear what you will get. That is why `===` was introduced. There is one exception which is `x == null` or `x != null` which checks for null or undefined.

Also here you can use the `startsWith` method instead of comparing `indexOf` to zero, which is more readable.

Reply:
I'm not sure if this is doing what we want for the filename but it will do to get started.